packages/components/src/components/input/input.styles.ts

```diff
@@ -12,6 +12,11 @@ export default css`
 	/* stylelint-disable */
   :host {
     display: block;
+
+    --syn-input-autofill-shadow: 0 0 0 var(--syn-input-height-large) var(--syn-input-background-color-hover) inset;
+    --syn-input-autofill-readonly-shadow: 0 0 0 var(--syn-input-height-large) var(--syn-input-readonly-background-color) inset;
+    --syn-input-autofill-text-fill-color: var(--syn-color-primary-500);
+    --syn-input-autofill-caret-color: var(--syn-input-autofill-caret-color);
```

Comment:
Currently the caret-color does not have a color, if the user does not set the `--syn-input-autofill-caret-color` variable.

The value of the variable should be the same as before the changes

```suggestion
    --syn-input-autofill-caret-color: var(--syn-input-color);
```

Reply:
It seems like the vendorism overwrites the var(--syn-input-color);  with var(--syn-input-autofill-caret-color); (see my other comment in the input.vendorism.ts)

Reply:
Fixed in latest